shepherd/api/views.py

```diff
@@ -133,30 +128,28 @@ def create_shepherd_routes(shepherd: Shepherd, storage: Storage) -> web.RouteTab
     @api.get("/jobs/{job_id}/input")
     @oapi.responds_with(ErrorResponse, code=404)
     @oapi.responds_with(FileResponse, code=200)
-    async def get_job_input(request: Request):
+    async def get_job_input(job_id: str, input_file: str = DEFAULT_PAYLOAD_FILE):
```

Comment:
Not sure about this, but I guess `input_file` should be also a path param

Reply:
Hm, sure. Is this the right PR to do it though?